<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="5" passed="3">
  <reporter-output>
    <line>
      <![CDATA[Test Case : friends---The following asserts failed:expected [true] but found [false]]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href="C:\Users\pankaj\workspace\Pageobject\screenshot\friends_Thu_Jun_04_17_40_59_IST_2020.jpg">Screenshot Captured</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href="C:\Users\pankaj\workspace\Pageobject\screenshot\friends_Thu_Jun_04_17_40_59_IST_2020.jpg"><img src="C:\Users\pankaj\workspace\Pageobject\screenshot\friends_Thu_Jun_04_17_40_59_IST_2020.jpg" height=400 width=400></a>]]>
    </line>
  </reporter-output>
  <suite name="Facebook Test" duration-ms="49650" started-at="2020-06-04T12:10:10Z" finished-at="2020-06-04T12:11:00Z">
    <groups>
    </groups>
    <test name="Check Facebook1" duration-ms="22927" started-at="2020-06-04T12:10:10Z" finished-at="2020-06-04T12:10:33Z">
      <class name="TestCases.LoginTest">
        <test-method status="PASS" signature="dologin(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestCases.LoginTest@574b560f]" name="dologin" duration-ms="22718" started-at="2020-06-04T17:40:10Z" data-provider="getdata" finished-at="2020-06-04T17:40:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[pankajakha@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[swat92#]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dologin -->
        <test-method status="SKIP" signature="dologin(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:TestCases.LoginTest@574b560f]" name="dologin" duration-ms="2" started-at="2020-06-04T17:40:33Z" data-provider="getdata" finished-at="2020-06-04T17:40:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[nitsysharma03@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[7097097.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[N]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping this exception]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping this exceptionat TestCases.LoginTest.dologin(LoginTest.java:41)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dologin -->
      </class> <!-- TestCases.LoginTest -->
    </test> <!-- Check Facebook1 -->
    <test name="Check Facebook2" duration-ms="26421" started-at="2020-06-04T12:10:33Z" finished-at="2020-06-04T12:11:00Z">
      <class name="TestCases.Searchfriends">
        <test-method status="PASS" signature="dologin(java.lang.String, java.lang.String)[pri:1, instance:TestCases.Searchfriends@44a7bfbc]" name="dologin" duration-ms="13861" started-at="2020-06-04T17:40:33Z" data-provider="getdata2" finished-at="2020-06-04T17:40:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[pankajakha@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[swat92#]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dologin -->
        <test-method status="PASS" signature="friends(java.lang.String)[pri:2, instance:TestCases.Searchfriends@44a7bfbc]" name="friends" duration-ms="1464" started-at="2020-06-04T17:40:47Z" data-provider="getvalue" finished-at="2020-06-04T17:40:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[par]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- friends -->
        <test-method status="FAIL" signature="friends(java.lang.String)[pri:2, instance:TestCases.Searchfriends@44a7bfbc]" name="friends" duration-ms="10251" started-at="2020-06-04T17:40:49Z" data-provider="getvalue" finished-at="2020-06-04T17:40:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[quw]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:expected [true] but found [false]at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)at TestCases.Searchfriends.friends(Searchfriends.java:34)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Test Case : friends---The following asserts failed:expected [true] but found [false]]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href="C:\Users\pankaj\workspace\Pageobject\screenshot\friends_Thu_Jun_04_17_40_59_IST_2020.jpg">Screenshot Captured</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href="C:\Users\pankaj\workspace\Pageobject\screenshot\friends_Thu_Jun_04_17_40_59_IST_2020.jpg"><img src="C:\Users\pankaj\workspace\Pageobject\screenshot\friends_Thu_Jun_04_17_40_59_IST_2020.jpg" height=400 width=400></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- friends -->
      </class> <!-- TestCases.Searchfriends -->
    </test> <!-- Check Facebook2 -->
  </suite> <!-- Facebook Test -->
</testng-results>
